website/docs/how-tos/how-to-query-tx-state.mdx

```diff
@@ -0,0 +1,103 @@
+---
+id: how-to-query-tx-state
+title: How to Query Transaction State
+---
+
+import Tooltip from "@components/Tooltip";
+
+# How to Query Transaction State
+
+Upon broadcasting a transaction with `send_transaction` RPC, you will receive a `txhash`, which only indicates that the transaction has been sent to the network, NOT verified on-chain.
+
+To query [transaction state](docs/tech-explanation/transaction.md#transaction-states), use `get_transaction`RPC. Remember, `send_transaction` operates asynchronously.
+
+For details on CKB JSON-RPC `send_transaction` method, visit the list of [CKB RPC methods](https://github.com/nervosnetwork/ckb/tree/develop/rpc#method-send_transaction).
+
+## Example: Query With JSON RPC
+
+### Request
+
+<details>
+    <summary>View details</summary>
+    ```javascript
+    {
+    "id": 42,
+    "jsonrpc": "2.0",
+    "method": "get_transaction",
+    "params": [
+        "0xa0ef4eb5f4ceeb08a4c8524d84c5da95dce2f608e0ca2ec8091191b0f330c6e3"
+    ]
+    }
+    ```
+</details>
+
+### Response
+
+<details>
```

Comment:
Can you use the **Tablist** component to put request and response in two different tabs?